tox.ini

```diff
@@ -30,8 +30,9 @@ commands =
 description = build readthedocs documentation
 changedir = {toxinidir}/docs
 commands =
-    pip install -r{toxinidir}/requirements/docs.txt  # TODO: Remove this once our Sphinx versions don't conflict
+    pip install codespell -r{toxinidir}/requirements/docs.txt  # TODO: Remove this once our Sphinx versions don't conflict
     sphinx-build -W --keep-going -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
+    codespell --ignore-words-list="ba,configurtion,ded,hel,perfom,wile" --skip="*.css,*.fjson,*.js,*.po,*.svg"
```

Comment:
Also, does codespell supports a config file, maybe (so we don't need to have these as arguments)?

Reply:
Looks like we can https://github.com/codespell-project/codespell/#using-a-config-file (.codespelrc)

Reply:
I always question the value of spreading state across many files especially when using tox which is designed to unify state for both local and CI-based testing.

Reply:
I think is fine, the file is autodetected, so is not like we always need to pass `--config`. The main advantage is that we can have comments over some options and we don't need to have this in one long line (for example we should have a comment about `configurtion` that is used as an example).